packages/chart/src/ChartUtils.ts

```diff
@@ -623,13 +653,27 @@ class ChartUtils {
     seriesVisibility: boolean | 'legendonly',
     theme = ChartTheme
   ): Partial<PlotData> {
-    const { name, plotStyle, lineColor, shapeColor, sources } = series;
+    const {
+      name,
+      isLinesVisible,
+      isShapesVisible,
```

Comment:
I think our types in the shim are slightly off for these. We have them as booleans when they should be optional booleans or `boolean | null`?

Reply:
Yup